test/pie-label.html

```diff
@@ -770,5 +771,37 @@
 
             });
         </script>
+
+        <script>
+            require([
+                'echarts'/*, 'map/js/china' */
+            ], function (echarts) {
+                const option = {
+                    series: [
+                        {
+                            name: 'Access From',
+                            type: 'pie',
+                            radius: 50,
+                            data: [
+                                { value: 1, name: 'Search Engine' },
+                                { value: 0, name: 'Direct' },
+                                { value: 0, name: 'Union Ads' },
+                                { value: 1, name: 'Video Ads' }
+                            ],
+                            label: {
+                                show: true
+                            },
+                        }
+                    ]
+                };
+
+                var chart = testHelper.create(echarts, 'main9', {
+                    title: 'the first label shoud be rendered correctly when it\'s followed by several zero values',
```

Comment:
You can reference this issue in the tittle so we can trace the origin of this case.

Reply:
@pissang It's done, please review again, thanks.